test/Romina.Api.UnitTests/Controllers/ProductControllerTest.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using Romina.Api.Controllers;
+using Romina.Api.Handlers;
+using Romina.Api.Models;
+using Romina.Api.Repositories;
+using Xunit;
+
+namespace Romina.Api.UnitTests.Controllers
+{
+    public class ProductControllerTest
+    {
+        private const string BasicId = "111";
+        private const string BasicMake = "Nike";
+        private const string BasicModel = "sport";
+
+        private readonly Mock<IProductRepository> _productRepository = new Mock<IProductRepository>();
+        private readonly Mock<IProductHandler> _productHandler = new Mock<IProductHandler>();
+
+        private Product _nikeTrouser = new Product
```

Comment:
do we need  _nikeTrouser, _nikeHat and _nikeGlove in these tests 

these products are used on lines 87-89 only, but we never use them to assert anything. So we can do without these and on line 93 simply return a `new List<Products>()`